lib/backup/notifier/mail.rb

```diff
@@ -230,17 +230,3 @@ def new_email
     end
   end
 end
-
-# Patch mail v2.5.4 Exim delivery method
-# https://github.com/backup/backup/issues/446
-# https://github.com/mikel/mail/pull/546
-module Mail
-  class Exim
-    def self.call(path, arguments, _destinations, encoded_message)
-      popen "#{path} #{arguments}" do |io|
-        io.puts encoded_message.to_lf
-        io.flush
-      end
-    end
-  end
-end
```

Comment:
Not sure what you mean. If I look at version 2.6.4 of mail I still see the same call to `#encoded`: https://github.com/mikel/mail/compare/2.6.3...2.6.4?expand=1#diff-70e36d3e2b37ee7aca19125cd388fd6fR47
The PR made by an earlier backup maintainer wasn't merged.